packages/google_mobile_maps/lib/src/bitmap.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of google_mobile_maps;
+
+class BitmapDescriptor {
```

Comment:
Perhaps worthy of a short dart-doc as the name is very generic.

Reply:
Done.